modules/@apostrophecms/schema/index.js

```diff
@@ -1709,7 +1710,6 @@ module.exports = {
         }
         const find = options.find;
         const builders = options.builders || {};
-        const hints = options.hints || {};
```

Comment:
Dead code removal. "Hints" were an A2 mechanism related to minSize and aspectRatio that we have chosen not to use this time, so I'm removing them before they wind up documented and supported.